src/Flowpipes/recipes.jl

```diff
@@ -280,6 +280,8 @@ end
         x_new, y_new = _plot_reachset_list(F, N, vars, ε, Nφ)
         append!(x, x_new)
         append!(y, y_new)
+        push!(x, N(NaN))
+        push!(y, N(NaN))
```

Comment:
Thanks!

I think we also need this change in line 345 below, could you add it?

Reply:
done